contract/scripts/propose-start-contract.sh

```diff
@@ -20,3 +20,15 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
 set +x # not so noisy for this part
 . /usr/src/upgrade-test-scripts/env_setup.sh
 voteLatestProposalAndWait
+
+sleep 2
```

Comment:
What is this sleep working around (I have PTSD anytime I see a delay in tests that is not looking for a trigger)

Reply:
`voteLatestProposalAndWait` finishes when the core-eval proposal passes. This gives it time to execute before we check to see if it works. IOU a comment.

The check was introduced a while ago without this delay, so I think it's not actually essential.

Reply:
Would it not be better to wait 2 blocks instead?

Reply:
ok